etl/pypasar/db/sql/postgres/provider/provider.sql

```diff
@@ -0,0 +1,45 @@
+-- *******************************************************************
+-- NAME: provider.sql
+-- DESC: Final table - provider
+-- *******************************************************************
+-- CHANGE LOG:
+-- DATE        VERS  INITIAL  CHANGE DESCRIPTION
+-- ----------  ----  -------  ----------------------------------------
+-- 2024-09-07  1.00           Initial create
+-- 2024-09-07  2.00           Insert data into provider table from staging view
+-- 2024-09-07 3.00            Updated the schema name
+-- *******************************************************************
+
+
+INSERT INTO {OMOP_SCHEMA}.provider
+(
+    provider_id,
+    provider_name,
+    npi,
+    dea,
+    specialty_concept_id,
+    care_site_id,
+    year_of_birth,
+    gender_concept_id,
+    provider_source_value,
+    specialty_source_value,
+    specialty_source_concept_id,
+    gender_source_value,
+    gender_source_concept_id
+)
+SELECT  provider_id,
+        NULL,
+        NULL,
+        NULL,
+        NULL,
+        NULL,
+        NULL,
+        NULL,
+        name,
```

Comment:
Some `provider_source_value` and `specialty_source_value` entries might not be distinct due to case sensitivity (e.g., GYNAECOLOGY vs. Gynaecology). It may be beneficial to ensure these values are comprehensively handled.

Reply:
Thanks, fixed it